### ISSUE TITLE
`dev` command fails in Git bash on Windows

### ISSUE THREAD

**Issue by jlengstorf** [gramps-graphql/gramps-cli#39]
Several things appear to be going wrong in the git bash shell on Windows. I'm unable to confirm any of these because I don't have a Windows machine, so this is just me repeating what I've been told.

## `del.sync` hangs infinitely

Despite the fact it _should_ be Windows-compatible, `del.sync()` seems to hang forever in git bash.

## `mkdirp` doesn't work

Similarly, `mkdirp` seems to be failing in git bash

## Relative paths are converted to absolute paths

The temporary directory path is being set to something like this:

```
C:\Users\dev\git\graphql-data-source-test\node_modules\@gramps\cli\dist\.tmp\C:\Users\dev\git\graphql-data-source-test
```

---

According to the dev having these issues, if `del.sync` is omitted, `mkdirp` is omitted, and a hard-coded temp dir path is used, the CLI works as intended.

@kbrandwijk, if you have a chance to take a look at this when you're free, I'd really appreciate it.

**Comment by cknabe** [comment]
$ node --version
v9.4.0

**Comment by jlengstorf** [comment]
More info:

This also happens using cmder.net.

The issues is happening on Windows 7.

**Comment by cknabe** [comment]
```const transpileDataSource = parentDir => dataSource => new Promise((resolve, reject) => {
  // const dirName = getDirName(dataSource);
  // const tmpDir = _path2.default.join(parentDir, dirName);
  const tmpDir = "C:/Users/IBM_ADMIN/git/graphql-data-source-softlayerintegration/npm_modules/@gramps/cli/dist/.tmp";

**Comment by cknabe** [comment]
```const makeParentTempDir = () => new Promise((resolve, reject) => {
  // cleanUpTempDir();

  // (0, _mkdirp2.default)(TEMP_DIR, err => {
  //   handleError(err, `Could not create ${TEMP_DIR}`);
  //   resolve(TEMP_DIR);
  // });
  resolve(TEMP_DIR);
});```

The above are the two changes that make it work for me.

**Comment by cknabe** [comment]
I've started looking into a solution

**Comment by jlengstorf** [comment]
Closed by #42
